modules/client-configuration/pages/manual-registration-overview.adoc

```diff
@@ -0,0 +1,11 @@
+[[manual-overview]]
+= Client Registration Overview
+
+There are two ways to register clients to your {productname} Server.
+Both methods work for either traditional or salt clients.
```

Comment:
The web ui variant work only for salt clients!

Traditional must be registered via bootstrap script.

Reply:
Oof, got it.